engine/src/main/java/org/terasology/engine/modes/StateMainMenu.java

```diff
@@ -121,6 +130,39 @@ public void init(GameEngine gameEngine) {
         if (!messageOnLoad.isEmpty()) {
             nuiManager.pushScreen(MessagePopup.ASSET_URI, MessagePopup.class).setMessage("Error", messageOnLoad);
         }
+
+        // TODO: enable it when exposing the telemetry to users
+        // pushLaunchPopup();
+    }
+
+    private void pushLaunchPopup() {
+        Config  config = context.get(Config.class);
```

Comment:
Two spaces between class name & instance name here. You can avoid this by autoformatting in whatever editor or IDE you're using (ideally this is IDEA ;) )